Source/Diagnostics/ParticleIO.cpp

```diff
@@ -127,11 +129,26 @@ MultiParticleContainer::WritePlotFile (const std::string& dir) const
 
             // Convert momentum to SI
             pc->ConvertUnits(ConvertDirection::WarpX_to_SI);
+
+            // build filter functors
```

Comment:
[draft until options are implemented, I guess] :-)

Reply:
yes, this PR will serve as suggestions for PR #703.